api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/DiagnosticReportHandler.java

```diff
@@ -1,12 +1,19 @@
 package org.openmrs.module.fhir.api.diagnosticreport;
 
 import ca.uhn.fhir.model.dstu2.resource.DiagnosticReport;
+import org.openmrs.Obs;
+
+import java.util.List;
+import java.util.Set;
 
 public interface DiagnosticReportHandler {
-	
-	public DiagnosticReport generateFHIRDiagnosticReport(DiagnosticReportTemplate omrsDiagnosticReport);
-	
-	public DiagnosticReportTemplate generateOpenMRSDiagnosticReport(DiagnosticReport fhirDiagnosticReport);
-	
+
 	public String getId();
```

Comment:
Why we need id in handler. It's coming from the FHIR diagnostic report. So we can remove it I think. Agree that we need those two methods as surnaga mentioned